plugin.yml

```diff
@@ -45,6 +45,15 @@ configuration:
       type: string
     default-secret-name:
       type: string
+    secret-store:
+      type: string
+    cluster-store:
+      type: string
+    external-secrets:
+      type: [string,array]
+    mount-path-external-secrets:
```

Comment:
`external-secrets-mount-path` makes more sense to me, but maybe you've got good reasons :-)

Reply:
Since the way buildkite does arrays (``external-secrets`` array becomes ``EXTERNAL_SECRETS_0``, ``EXTERNAL_SECRETS_1`` ... , I used the design pattern commonly used for other arrays where it gets everything starting with ``EXTERNAL_SECRETS_``, and ``external-secrets-mount-path`` gets picked up by that. 

I really with buildkite didn't work like that, but oh well. I would much rather it be ``mount-path-external-secrets`` too.

Reply:
Gotcha, makes sense given those semantics. Thanks for the clarification!